spring-web/src/main/java/org/springframework/web/server/ServerWebInputTooLargeException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2002-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.web.server;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.lang.Nullable;
+
+/**
+ * Exception for errors that fit response status 413 (payload too large) for use in
+ * Spring Web applications.
+ *
+ * @author Kim Bosung
+ * @since 5.0
```

Comment:
```suggestion
 * @since 6.2
```
since 6.2 is correct?